third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import com.google.common.collect.ImmutableSet;
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.CustomPassExecutionTime;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+  
+  ImmutableSet<String> suffixTypes = ImmutableSet.of(
+      "log.dev.fine",
+      "log.dev.assert"
+      );
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    AmpPass ampPass = new AmpPass(getCompiler(), suffixTypes);
+    options.addCustomPass(CustomPassExecutionTime.BEFORE_OPTIMIZATIONS, ampPass);
+    return options;
+  }
+
+  public static void main(String[] args) {
+    AmpCommandLineRunner runner = new AmpCommandLineRunner(args);
+    if (runner.shouldRunCompiler()) {
+      runner.run();
+    }
+    if (runner.hasErrors()) {
+      System.exit(-1);
```

Comment:
this is an exact copy of the main code in the super class, and in my testing the errors/warning do get printed out.